hele-logo/src/index.js

```diff
@@ -26,7 +26,6 @@ class HeleLogo extends Component {
             src="https://www.hele.cz/assets/img/logo-hele-bubble.png"
             href="https://www.hele.cz"
             padding="10"
-            alt="Hele logo"
```

Comment:
alt="" bude lepší

http://www.w3schools.com/tags/att_img_alt.asp

- Use alt="" if the image is only for decoration